### DISCUSSION TITLE
post/homelab-journey-1/

### DISCUSSION THREAD

**Discussion by giscus** [ecwu/ecwu.github.io#7] [Blog Post Discussion]
# post/homelab-journey-1/

Homepage and Blog of ECWU

https://ecwuuuuu.com/post/homelab-journey-1/

**Comment by TerenceLiu98** [comment]
想搭建一个类似的 Homelab 需要多少资源比较合适呢 :) 现在手头上只有几台 R7525 和 H100 🥹

**Reply by ecwu** [reply]
建议送我 :-)
